### PR TITLE
update dp60 configuration

### PR DESCRIPTION
**Pull request by yulei**
## Description
turn on  rgb underglow and rgb matrix at same time

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dp60/keymaps/allleds/rules.mk [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# Do not put the microcontroller into power saving mode
+# when we get USB suspend event. We want it to keep updating
+# backlight effects.
+#OPT_DEFS += -DNO_SUSPEND_POWER_DOWN
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite # Virtual DIP switch configuration(+1000)
```
Comment (fauxpark) [inline keyboards/dp60/keymaps/allleds/rules.mk]:
```suggestion
BOOTMAGIC_ENABLE = lite # Virtual DIP switch configuration
```

File: keyboards/dp60/keymaps/allleds/rules.mk [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+MOUSEKEY_ENABLE = no        # Mouse keys(+4700)
```
Comment (fauxpark) [inline keyboards/dp60/keymaps/allleds/rules.mk]:
```suggestion
MOUSEKEY_ENABLE = no        # Mouse keys
```

File: keyboards/dp60/keymaps/allleds/rules.mk [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+EXTRAKEY_ENABLE = yes # Audio control and System control(+450)
```
Comment (fauxpark) [inline keyboards/dp60/keymaps/allleds/rules.mk]:
```suggestion
EXTRAKEY_ENABLE = yes # Audio control and System control
```

File: keyboards/dp60/keymaps/allleds/rules.mk [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+CONSOLE_ENABLE = no         # Console for debug(+400)
```
Comment (fauxpark) [inline keyboards/dp60/keymaps/allleds/rules.mk]:
```suggestion
CONSOLE_ENABLE = no         # Console for debug
```

File: keyboards/dp60/keymaps/allleds/rules.mk [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
```
Comment (fauxpark) [inline keyboards/dp60/keymaps/allleds/rules.mk]:
```suggestion
MIDI_ENABLE = no            # MIDI support
```

File: keyboards/dp60/keymaps/allleds/rules.mk [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+
+# Do not put the microcontroller into power saving mode
+# when we get USB suspend event. We want it to keep updating
+# backlight effects.
+#OPT_DEFS += -DNO_SUSPEND_POWER_DOWN
```
Comment (fauxpark) [inline keyboards/dp60/keymaps/allleds/rules.mk]: Please remove this block.

Comment (yulei) [reply keyboards/dp60/keymaps/allleds/rules.mk]: ok, removed

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**